src/components/_myHead.tsx

```diff
@@ -0,0 +1,40 @@
+import Head from "next/head";
+import { NextPage } from "next";
+
+interface MyHeadProps {
+  title?: string;
+  description?: string;
+}
+
+const MyHead: NextPage<MyHeadProps> = ({ title, description }) => {
```

Comment:
これは、ページコンポーネントではないので、 `NextPage` は使うべきではないですね。
（動くけどコードの意味的におかしい）

```suggestion
const MyHead = ({ title, description = "勤怠管理アプリ" }: MyHeadProps) => {
```

Reply:
https://github.com/tatata05/next-app/pull/2/commits/d7e533e21c6719d03c0af2d217c03ebcb03b3274
NextPageを使用しない方法に変更しました！